src/sidebar/components/ThreadList.tsx

```diff
@@ -209,17 +211,20 @@ export default function ThreadList({ threads }: ThreadListProps) {
     }
   }, [store, newAnnotationTag]);
 
-  const editing = store.countDrafts() > 0;
-  const highlightedAnnotations = store.highlightedAnnotations();
+  const mostRecentlyHighlightedAnnotationId = useMemo(
+    // If multiple highlighted annotations exist, assume that the last one in
+    // the list is the most recent.
+    () => highlightedAnnotations[highlightedAnnotations.length - 1],
+    [highlightedAnnotations],
+  );
```

Comment:
I think it would be better to explicitly sort the annotations here, since the `highlightedAnnotations` store module doesn't promise anything about the order. For unsaved annotations we ought to do the same, provided that the date field is actually populated. I can't remember if it is.